doc/source/cheat_sheet/cheat_sheet.qmd

```diff
@@ -0,0 +1,278 @@
+---
+title: PyMAPDL cheat sheet
+format: cheat_sheet-pdf
+version: main
+footer: PyMAPDL
+footerlinks:
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/'
+    text: Documentation
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/getting_started/index.html'
+    text: Getting started
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/examples.html'
+    text: Examples
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/api/index.html'
+    text: API reference
+  - urls: 'https://github.com/ansys/pymapdl/discussions'
+    text: Discussions
+  - urls: 'https://github.com/ansys/pmapdl/issues'
+    text: 'Issues'
+execute:
+    output: false
+    eval: true
```

Comment:
@germa89 all the code blocks are running and evaluation when the cheatsheet is building by putting this flag ` eval: true`

Reply:
The only ones that are not tested are the one with the ``#| eval: false`` mention at the beginning of the cell.

Reply:
LGTM then.